tests/test_lsst.py

```diff
@@ -0,0 +1,928 @@
+from __future__ import with_statement
+import unittest
+import numpy as np
+import warnings
+import os
+import galsim
+from galsim.lsst import LsstCamera, LsstWCS, _nativeLonLatFromRaDec
```

Comment:
This can't be at the top level.  We can't require the LSST stack for regular runs of nosetests (via `scons tests`).  So there needs to be some graceful way to ignore the ImportErrors if someone is running `scons tests` and doesn't have the stack installed.  OTOH, we probably want to let those errors through when someone is running this from the command line.  So probably something like
```
try:
    from galsim.lsst import LsstCamera, LsstWCS, _nativeLonLatFromRaDec
except:
    if __name__ != '__main__':
        print "No LSST stack detected.  Skipping tests in test_lsst.py"
        no_stack=True
    else:
        raise
else:
    no_stack=False
```
then in each test something like
```
if no_stack: return
```
so they gracefully ignore the lack of an installed LSST stack.